challenges/intermediate/find-the-neighbour/solution.js

```diff
@@ -0,0 +1,58 @@
+// https://jsfiddle.net/dianaberna/bmgypt78/
+const regexCube = /-\||[a-zA-Z]+/gm;
+
+const cubeStr = `
+   ______________________
+  /_____________________/|
+ /_____________________/ |
+ |      |      |      |  |
+ | Tom  |Jerry | John | /|
+ |______|______|______|/ |
+ |      |      |      |  |
+ | Mike |Peter |Alice | /|
+ |______|______|______|/ |
+ |      |      |      |  |
+ |  Bob | Bill | Wang | /
+ |______|______|______|/
+`;
+
+
+function find_neighbour(cube, query) {
+  var nameList = [];
+  var queryName = query.split(":")[0];
+  while((array = regexCube.exec(cube)) !== null){
+    	nameList.push(array[0])
+  }
+  
+  let arrayMovement = [
```

Comment:
I get the idea behind this `arrayMovement` array, but I'd suggest you avoiding the usage of the `nope` property, as it would fail if we add another column/row